infrastructure/helm/quantumserverless/values.yaml

```diff
@@ -107,7 +107,23 @@ ray-cluster:
   head:
     rayStartParams:
       dashboard-host: '0.0.0.0'
-
+    ports: [{containerPort: 10001, name: client},{containerPort: 6379, name: redis},{containerPort: 8265, name: dashboard},{containerPort: 8080, name: metrics},{containerPort: 8000, name: serve},{containerPort: 4180, name: proxy}]
+    sidecarContainers:
+        - name: oauth-proxy
+          image: quay.io/oauth2-proxy/oauth2-proxy:v7.3.0
+          imagePullPolicy: IfNotPresent
+          args:
```

Comment:
the args are visible in the cluster via `kubectl get/describe po` ... does that matter for client/cookie secrets?